content/news/playtest-20150408.md

```diff
@@ -0,0 +1,17 @@
+We have finalized the balancing for the Red Alert mod, so it's time for another playtest!
+
+The changes this time around include:
+
+  - More balancing adjustments in Red Alert. Please see [the changelog](http://changelog.openra.net) for a detailed list of changes.
+  - Fixed enemies erroneously getting an "(Ally)" suffix in chat messages.
+  - Password protected games will be indicated as such in the server browser even when the game has already started.
```

Comment:
You could shorten this to "Games in progress that are password protected are now indicated as such in the server browser." so that it doesn't wrap to a second line.